### DISCUSSION TITLE
Adguard home suddenly asking for username and password?

### DISCUSSION THREAD

**Discussion by bobloadmire** [AdguardTeam/AdGuardHome#7271] [Q&A]
I've never had to login before. I always login locally using local IP, but randomly now its asking for username and pw. I never set one up. Anyone know where to even begin here?

**Comment by schzhn** [comment]
The password is set the first time you start AGH. If you forget it, you can to reset it by generating a hash of the new password and editing the  `users.password` property in the configuration file.
```
mkpasswd -m bcrypt -R 10 <super-strong-password>
```
Related #6865
